examples/netlify-functions/functions/addToCart/addToCart.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
```

Comment:
In the interest of avoiding unnecessary nesting in paths, let's change this file from `examples/netlify-functions/functions/addToCart/addToCart.js` to just `examples/netlify-functions/functions/addToCart.js` . I've done that with other projects and it should work.